README.md

```diff
@@ -79,11 +79,11 @@ See [Examples Hub README page](Documentation/README_ExampleHub.md) to learn abou
 [![Example Scene](Documentation/Images/MRTK_ExamplesHub.png)](Documentation/README_HandInteractionExamples.md)
 
 
-# Sample App
+# Sample apps made with MRTK
+| [![Periodic Table of the Elements](Documentation/Images/MRDL_PeriodicTable.jpg)](https://medium.com/@dongyoonpark/bringing-the-periodic-table-of-the-elements-app-to-hololens-2-with-mrtk-v2-a6e3d8362158)| [![Periodic Table of the Elements](Documentation/Images/MRTK_GalaxyExplorer.jpg)](https://docs.microsoft.com/en-us/windows/mixed-reality/galaxy-explorer-update)|
```

Comment:
Super nit: The two images are SLIGHTLY different sizes from each other. My OCD is somewhat strong but here's what I mean:

![image](https://user-images.githubusercontent.com/5840182/69262572-93b03080-0b78-11ea-8b29-491f30e64714.png)

Reply:
@wiwei Acutally, they have the same size but this is a limitation of the GitHub's markdown table :) There is no way to control the width of the table and image width. It is controlled by the amount of the text in each column.